_docs/concepts/auth.md

```diff
@@ -0,0 +1,138 @@
+---
+category: Concepts
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
+parent: Network and Auth
+order: 20
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+The page explains Auth's role and general architecture.
+
+
+## Istio Auth Concept
+
+*Status: Draft*
+
+*Date: April, 2017*
+
+
+[TOC]
+
+
+### Overview {#overview}
+
+Istio Auth aims at enhancing the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Presenting a strong identity that represents the role of the service to enable interoperability across clusters and clouds.
+*   Securing the service to service communication and the end-user to service communication.
+*   Providing a key management system to automate key/cert generation, distribution, rotation, and revocation.
+*   Upcoming features:
```

Comment:
I'd turn this into a separate list, because it isn't currently responsible for either of this.

In future versions it will also provide:

* fine-grained...
* powerful authorization..

Or you could leave it out as it's already in the Future Work section below? That might be better as it means the doc only describes what it actually does now.